src/source.ts

```diff
@@ -136,6 +136,21 @@ export interface SourceConfig extends NativeInstanceConfig {
   analyticsSourceMetadata?: SourceMetadata;
 }
 
+/**
+ * The remote playback config for a source.
+ * @platform iOS
+ */
+export interface SourceRemotePlaybackConfig {
```

Comment:
NIT: The naming might be a bit problematic as it suggests that it has something to to with a remote `PlaybackConfig`. Is the idea that this will in the future potentially contain more than just the `castSourceConfig`?

Reply:
I also struggled to find a name, this was like the 3rd one I used.
I actually like `SourceRemoteControlConfig` name, will change it to that.

Reply:
Renamed in 852d26f